### ISSUE TITLE
loss function 

### ISSUE THREAD

**Issue by leezhibo** [BitTiger-MP/DS502-AI-Engineer#28]
<img width="664" alt="second_" src="https://user-images.githubusercontent.com/29693214/28448421-bec42ed4-6d8c-11e7-85f4-b981334ce04c.png">
<img width="704" alt="first_" src="https://user-images.githubusercontent.com/29693214/28448425-c5cb3178-6d8c-11e7-8d66-39dc438c2c9e.png">
请问在答案中，softmax regression练习和mlp 练习中用的loss function不太一样呢，mlp中用的也是softmax 做最后一层，但是却比soft Max regression 练习中多了一个(1 - y)
